tests/portfind_test.py

```
@@ -1,12 +1,80 @@
+
+    result = subprocess.run([
+        sys.executable, '-m', 'src.portfinder.cli',
+        '127.0.0.1', '--start', '1', '--end', '1'
+    ], capture_output=True, text=True)
+    assert result.returncode == 0
+    assert 'Please wait, scanning' in result.stdout
+
+@pytest.mark.asyncio
+async def test_scan_port():
+    result = await scan_port("127.0.0.1", 5000, DEFAULT_TIMEOUT)
+    assert result is 5000
```

Ruff (0.12.2)

Use `==` to compare constant literals

Replace `is` with `==`

(F632)